### PR TITLE
Integrate e2e-test-cypress job with Buildmaster

### PR DESCRIPTION
**Pull request by rgagarinr**
## Summary:
This PR runs the cypress job in parallel with the existing e2e-test job.

Issue: FEI-4480

## Test plan:
Run the script with replay on both scripts

### CODE REVIEW

File: jobs/e2e-test-cypress.groovy [hunk 1]
```diff
@@ -72,7 +72,10 @@ Defaults to GIT_REVISION.""",
 ).apply()
 
 // We use the build name as a unique identifier for user notifications. 
-BUILD_NAME = "e2e-test #${env.BUILD_NUMBER} (${params.URL}: ${params.CYPRESS_GIT_REVISION})"
+BUILD_NAME = "build e2e-test #${env.BUILD_NUMBER} (${params.URL}: ${params.REVISION_DESCRIPTION})"
```
Comment (csilvers) [inline jobs/e2e-test-cypress.groovy:75]: Do you want to put "cypress" in here somewhere?

**Review by csilvers — APPROVED** [review]
Awesome!  Fingers crossed this works well.

File: jobs/e2e-test-cypress.groovy (hunk 1 above)
Comment (rgagarinr) [reply jobs/e2e-test-cypress.groovy:75]: Yeah, maybe something like "e2e-cypress-test"

**Comment by rgagarinr** [comment]
Thanks @csilvers for all your help with this stuff!

**Review by dbraley — COMMENTED** [review]
👍🏻

**Pull request merged**